src/fragment-components/a-link.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { Link, Checkbox, TextInput } from 'carbon-components-react';
+import { css } from 'emotion';
+import { AComponent, ComponentInfo } from './a-component';
+import { ComponentCssClassSelector } from '../components/css-class-selector';
+
+import image from './../assets/component-icons/link.svg';
+import { 
+	angularClassNamesFromComponentObj, 
+	nameStringToVariableString, 
+	reactClassNamesFromComponentObj 
+} from '../utils/fragment-tools';
+
+export const ALinkStyleUI = ({selectedComponent, setComponent}: any) => {
+	return <>
+		<TextInput
+			value={selectedComponent.text}
+			labelText='Text'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					text: event.currentTarget.value
+				});
+			}}
+		/>
+		<Checkbox
+			labelText='Disabled'
+			id='disable-label'
+			checked={selectedComponent.disabled}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					disabled: checked
+				});
+			}}
+		/>
+		<Checkbox
+			labelText='Inline'
+			id='Inline-select'
+			checked={selectedComponent.inline}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					inline: checked
+				});
+			}}
+		/>
+		<ComponentCssClassSelector componentObj={selectedComponent} setComponent={setComponent} />
+	</>
+};
+
+export const ALinkCodeUI = ({selectedComponent, setComponent}: any) => {
+	return <>
+		<TextInput
+			value={selectedComponent.codeContext?.name}
+			labelText='Input name'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					codeContext: {
+						...selectedComponent.codeContext,
+						name: event.currentTarget.value
+					}
+				});
+			}}
+		/>
+		<TextInput
+			value={selectedComponent.codeContext?.href || ''}
+			labelText='href'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					codeContext: {
+						...selectedComponent.codeContext,
+						href: event.currentTarget.value
+					}
+				});
+			}}
+		/>
+	</>
+};
+
+export const ALink = ({
+	children,
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		headingCss={css`display: inline-block;margin-top: -40px;margin-left: 54px;min-width: 83px;`}
+		className={css`align-items: center; justify-content: center; display: inline-flex; cursor: pointer;`}
```

Comment:
DO we really need these?

Reply:
We aren't using any headingCss/Classes for AText component - so not sure why it's an issue here 🤔

Reply:
```suggestion
```
I think it is safe to remove since the wrapper with the bin appears exactly on top for me, hopefully it is the same case for you. Removing the headingCss & class should be ok

Reply:
removinfg these classes results in this defect https://github.com/IBM/carbon-components-builder/issues/61

Reply:
Hmm that is interesting... so I cloned your PR & removed it and works for me after I removed this code 🤔
What is your screen resolution? What browser are you using?

![image](https://user-images.githubusercontent.com/38994122/159035404-dd3dcb5c-45c8-4c4b-8bba-d6604036bf4d.png)